golem_messages/settings.py

```diff
@@ -4,6 +4,7 @@
 
 MSG_TTL = datetime.timedelta(minutes=10)
 FUTURE_TIME_TOLERANCE = datetime.timedelta(minutes=5)
+PAYMENT_TIMESTAMP_TOLERANCE = datetime.timedelta(minutes=15).total_seconds()
```

Comment:
Since all tolerances are `timedelta`s I think one should also be a `timedelta`

Reply:
Done